src/components/Tabs/Tabs.tsx

```diff
@@ -0,0 +1,74 @@
+import styles from '@/components/Tabs/tabs.module.scss';
+import { useAppContext } from '@/context';
+import { isUserAuthorizedContext } from '@/context/isUserAuthorized';
+import { useTabs } from '@/hooks/useTabs';
+import { useContext } from 'react';
+import TaskList from '../tasks/TaskList/TaskList';
+import task from '@/interfaces/task.type';
+import fetch from '@/helperFunctions/fetch';
+import { toast, ToastTypes } from '@/helperFunctions/toast';
+import { IN_PROGRESS } from '@/components/constants/task-status';
+import { TASKS_URL } from '../constants/url';
+const { SUCCESS, ERROR } = ToastTypes;
+
+type TabsProps = {
+  filteredTasks: task[] | any
+  title: string
+};
+
+async function updateCardContent(id: string, cardDetails: task) {
+  try {
+    const { requestPromise } = fetch({
+      url: `${TASKS_URL}/${id}`,
+      method: 'patch',
+      data: cardDetails,
+    });
+    await requestPromise;
+    toast(SUCCESS, 'Changes have been saved !');
+  } catch (err: any) {
+    if ('response' in err) {
```

Comment:
```js
const message = err?.response?.data?.message ?? err.message;
```
OR create and use a utility function to get error message.
```js
function getErrorMessage(err)
{
if(err.response.message)
{
return err.response.data.message;
}
return err?.message ?? 'Something Went Wrong'
}

Reply:
@Maheima you can use the first method which @shmbajaj has explained and then pass it to toast you don't need to return as well